packages/fx-core/resource/package.nls.json

```diff
@@ -201,6 +202,22 @@
   "plugins.sql.errorMessage.ServicePrincipalWarning": "service principal admin in azure sql can't add database user <%s>. You can add the user for %s manually",
   "plugins.sql.errorMessage.DomainError": "Conditional Access policy requires a compliant device, and the device is not compliant. %s",
   "plugins.sql.errorMessage.GuestAdminError": "SQL admin does not have enough permission to add database user. %s",
+  "plugins.cicd.ciOption.detail": "Help checkout code, build and run test.",
+  "plugins.cicd.cdOption.detail": "Help checkout code, build, test and deploy to cloud.",
+  "plugins.cicd.provisionOption.detail": "Help create/update resources in cloud and Teams app registration.",
+  "plugins.cicd.publishOption.detail": "Help publish Teams app to tenants.",
+  "plugins.cicd.whichProvider.title": "Select a CI/CD Provider",
+  "plugins.cicd.whichTemplate.title": "Select template(s)",
+  "plugins.cicd.whichEnvironment.title": "Select an environment",
+  "plugins.cicd.ProgressBar.scaffold.title": "Scaffolding workflow automation files",
```

Comment:
Seems we do not need to localize progress bar.

Reply:
Interesting - https://github.com/OfficeDev/TeamsFx/blob/dev/packages/fx-core/src/plugins/resource/frontend/resources/messages.ts this file from frontend didn't do localization and it's expected?

Reply:
Just synced with @HuihuiWu-Microsoft , strings in progressBar are expected to be localized.